.github/ISSUE_TEMPLATE/issue.md

```diff
@@ -0,0 +1,18 @@
+---
+name: Issue
+about: Generic issue template for a proposed OpenAssetIO change
+title: Issue Title
+labels: ''
+assignees: ''
+
+---
+
+## What
+
+## Why
+
```

Comment:
[consider] Can we add something that explains what is expected here, eg:

"Describe why, in terms of a a user story, etc..."

Reply:
[96524dd](https://github.com/OpenAssetIO/OpenAssetIO/pull/773/commits/96524ddb806b4be2530b03019b8d532471ebb19b)

I'd favour avoiding the user story framing, we don't tend to conform to them tightly (wisely imo,) and i'd like to avoid the dissonance there.